conjureplugin/backcompat/types.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2025 Palantir Technologies. All rights reserved.
```

Comment:
This can be done in a follow-up PR, but this package should probably export an interface and helpers that streamlines the process of implementing a `backcompat` asset and provides the logic that would be common to all of them (for example, parsing the JSON input)